docs/use-cases/inferencing/README.md

```diff
@@ -0,0 +1,153 @@
+|                                | Persistent Volume | Tunables(`read_ahead_kb` etc) are in the PV's manifest                                       |
+|                                | NFS Volume        | A privileged initContainer is needed to tune performance                                     |
+| Model Update                   | GCSFuse           | just copy new model weights to GCS bucket                                                    |
+|                                | Persistent Volume | an extra workflow is needed                                                                  |
+|                                | NFS Volume        | just copy new model weights to NFS                                                           |
+| Performance                    |                   |                                                                                              |
+| Warm-up                        | GCSFuse           | Warm-up needed                                                                               |
+|                                | Persistent Volume | No warm-up                                                                                   |
+|                                | NFS Volume        | No warm-up                                                                                   |
+| Max read throughput            | GCSFuse           | Depending on node disk type and size                                                         |
+|                                | Persistent Volume | Depending on volume type and size                                                            |
+|                                | NFS Volume        | Depending on FileStore volume                                                                |
+| Price                          |                   |                                                                                              |
+| Over provisioning requirements | GCSFuse           | Node disk over-provisioning needed. May require additional compute memory for larger models. |
+|                                | Persistent Volume | Over-provisioning is shared by up to 100 nodes                                               |
+|                                | NFS Volume        | Over-provisioning can be shared by other applications                                        |
+| Cost summary\*                 | GCSFuse           | $$\*\*                                                                                       |
+|                                | Persistent Volume | $                                                                                            |
+|                                | NFS Volume        | $$$                                                                                          |
+
+\* This will vary based on your implementation
+
+\*\* for larger LLM’s i.e. 70B+ parameter
```

Comment:
There is one of the strange single quotes here, can you change it to a standard single quote.